src/persist-client/src/write.rs

```diff
@@ -170,9 +170,17 @@ where
         upper: Antichain<T>,
         last_heartbeat: EpochMillis,
     ) -> Self {
-        let debug_state = HandleDebugState {
-            hostname: cfg.hostname.to_owned(),
-            purpose: purpose.to_owned(),
+        let writer_config = WriterConfig {
+            writer_id: writer_id.clone(),
+            lease_duration_ms: cfg
```

Comment:
little bit of a bummer for this to be snapshotted at the time the handle is created instead of reflecting the latest value each time we heartbeat/CaA. thoughts on cloning the `DynamicConfig` into `WriterConfig` and having a `fn lease_duration_ms` that grabs and returns the latest value?

Reply:
Doable, but I think it pushes this particular change into "not worth it" territory for me... the benefits of this commit are pretty marginal (bundling params to simplify test code) so if we're adding new complexity / overhead it's a bit of a wash. I'd prefer to drop the lease duration from the writer config entirely, I think... but maybe even more tempted to just drop the refactoring.

Reply:
> I'd prefer to drop the lease duration from the writer config entirely, I think... but maybe even more tempted to just drop the refactoring.

either of these are also fine by me